packages/xrpl/package.json

```diff
@@ -56,7 +56,7 @@
     "build:lib": "tsc --build tsconfig.build.json",
     "build:web": "webpack",
     "build:browserTests": "webpack --config ./test/webpack.config.js",
-    "analyze": "run-s build:web --analyze",
+    "analyze": "webpack --analyze",
```

Comment:
why is this in the PR?

Reply:
It fixes the webpack bundle size tool which i used to weigh whether or not this change was worth it.